src/utils/alloyCookie.js

```diff
@@ -0,0 +1,59 @@
+import cookie from "./cookie";
+
+const COOKIE_NAME = "TEMP_ALLOY_COOKIE";
+const COOKIE_EXPIRY = 7;
+
+const getKeyValPair = pipedString => {
+  const keyVals = {};
+  pipedString.replace(/([^=|]+)\|([^|]*)/g, (match, key, val) => {
+    keyVals[key] = val;
+  });
+  return keyVals;
+};
+
+const createPipedString = keyVals => {
```

Comment:
This should probably be its own utility.

Reply:
It will work only when the  cookie format is "KEY1|VAL1|KEY2|VAL2|KEY23|VAL3", 
Do you still think it should be a separate util

Reply:
I have moved it as we discussed, now it is a JSON string